react/src/components/FileBrowserButton.tsx

```diff
@@ -100,17 +130,25 @@ const FileBrowserButton: React.FC<FileBrowserButtonProps> = ({
               }
               if (results?.rejected && results.rejected.length > 0) {
                 const error = results.rejected[0].reason;
-                modal.error({
-                  title: error?.title,
-                  content: getErrorMessage(error),
-                });
+                if (_.includes(error.message, 'DUPLICATED_SESSION')) {
+                  upsertNotification({
+                    key: `sftp-${vfolder?.row_id}`,
```

Comment:
```suggestion
                    key: `filebrowser-${vfolder?.row_id}`,
```

The sftp and filebrowser notifications have same key. Please separate them.